data.go

```diff
@@ -4,19 +4,32 @@ import (
 	"io"
 )
 
+type EnhancedCode [3]int
+
 // SMTPError specifies the error code and message that needs to be returned to the client
 type SMTPError struct {
-	Code    int
-	Message string
+	Code         int
+	EnhancedCode EnhancedCode
+	Message      string
 }
 
+// NoEnhancedCode is used to indicate that enhanced error code should not be
+// included in response.
```

Comment:
Can you clarify that this is for use by extensions, and that people should likely use `EnhancedCodeNotSet` instead?

Reply:
d37e6b0